src/shogun/regression/LeastAngleRegression.cpp

```diff
@@ -175,75 +193,76 @@ bool CLeastAngleRegression::train_machine(CFeatures* data)
 	//========================================
 	int32_t nloop=0;
 	while (m_num_active < max_active_allowed && max_corr/n_vec > m_epsilon && !stop_cond)
-	{
+	{			
 		// corr = X' * (y-mu) = - X'*mu + Xy
-		Map<VectorXd> map_corr(&corr[0], n_fea);
-		Map<VectorXd> map_mu(&mu[0], n_vec);
+		typename SGVector<ST>::EigenVectorXtMap map_corr(&corr[0], n_fea);
+		typename SGVector<ST>::EigenVectorXtMap map_mu(&mu[0], n_vec);
 		
 		map_corr = map_Xy - (map_Xr*map_mu);
 		
 		// corr_sign = sign(corr)
-		for (uint32_t i=0; i < corr.size(); ++i)
+		for (index_t i=0; i < corr.size(); ++i)
 			corr_sign[i] = CMath::sign(corr[i]);
 
 		// find max absolute correlation in inactive set
 		find_max_abs(corr, m_is_active, i_max_corr, max_corr);
-
+		
```

Comment:
can you check your PR to avoid such whitespace changes, in particular if you just add random tabs to lines